peers-lib/src/main/java/net/sourceforge/peers/media/DtmfFactory.java

```diff
@@ -53,6 +53,7 @@
         rtpPacket.setMarker(true);
         packets.add(rtpPacket);
 
+		/*

```

Comment:
can you remove commented out lines and use the right indentation for the removed loop?

thanks